src/gfnff/gffcom.f90

```diff
@@ -0,0 +1,165 @@
+     &1,1,                                         1, 0, 0, 0, 0, 0,&!Ar
+     &1,1,2,          2, 2, 2, 2, 2, 2, 2, 2, 2,   1, 0, 0, 0, 0, 0,&!Kr
+     &1,2,2,          2, 2, 2, 2, 2, 2, 2, 2, 2,   1, 1, 0, 0, 0, 0,&!Xe
+     &1,2,2,  14*2,   2, 2, 2, 2, 2, 2, 2, 2, 2,   1, 1, 1, 1, 0, 0/ !Rn  ! At is NOT a metal, Po is borderline but slightly better as metal
+      data group / &
+     &1,                                                          8,&!He
+     &1,2,                                         3, 4, 5, 6, 7, 8,&!Ne
+     &1,2,                                         3, 4, 5, 6, 7, 8,&!Ar
+     &1,2,-3,        -4,-5,-6,-7,-8,-9,-10,-11,-12,3, 4, 5, 6, 7, 8,&!Kr
+     &1,2,-3,        -4,-5,-6,-7,-8,-9,-10,-11,-12,3, 4, 5, 6, 7, 8,&!Xe
+     &1,2,-3, 14*-3, -4,-5,-6,-7,-8,-9,-10,-11,-12,3, 4, 5, 6, 7, 8/ !Rn
+      data normcn/ &  ! only for non metals well defined
+     &1,                                                          0,&!He
+     &4,4,                                         4, 4, 4, 2, 1, 0,&!Ne
+     &4,4,                                         4, 4, 4, 2, 1, 0,&!Ar
+     &4,4,4,          4, 6, 6, 6, 6, 6, 6, 4, 4,   4, 4, 4, 4, 1, 0,&!Kr
+     &4,4,4,          4, 6, 6, 6, 6, 6, 6, 4, 4,   4, 4, 4, 4, 1, 0,&!Xe
+     &4,4,4,  14*4,   4, 6, 6, 6, 6, 6, 6, 6, 4,   4, 4, 4, 4, 1, 0/ !Rn
+      data repz  / &
+     &1.,                                                         2.,&!He
+     &1.,2.,                                       3.,4.,5.,6.,7.,8.,&!Ne
+     &1.,2.,                                       3.,4.,5.,6.,7.,8.,&!Ar
+     &1.,2.,3.,      4.,5.,6.,7.,8.,9.,10.,11.,12.,3.,4.,5.,6.,7.,8.,&!Kr
+     &1.,2.,3.,      4.,5.,6.,7.,8.,9.,10.,11.,12.,3.,4.,5.,6.,7.,8.,&!Xe
+     &1.,2.,3.,14*3.,4.,5.,6.,7.,8.,9.,10.,11.,12.,3.,4.,5.,6.,7.,8./ !Rn
+
+     !Mass
+     real(wp)              :: ams(107)
+     data  ams /  1.00790d0,  4.00260d0,  6.94000d0,  9.01218d0,&
+     &10.81000d0, 12.01100d0, 14.00670d0, 15.99940d0, 18.99840d0,&
+     &20.17900d0, 22.98977d0, 24.30500d0, 26.98154d0, 28.08550d0,&
+     &30.97376d0, 32.06000d0, 35.45300d0, 39.94800d0, 39.09830d0,&
+     &40.08000d0, 44.95590d0, 47.90000d0, 50.94150d0, 51.99600d0,&
+     &54.93800d0, 55.84700d0, 58.93320d0, 58.71000d0, 63.54600d0,&
+     &65.38000d0, 69.73500d0, 72.59000d0, 74.92160d0, 78.96000d0,&
+     &79.90400d0, 83.80000d0, 85.46780d0, 87.62000d0, 88.90590d0,&
+     &91.22000d0, 92.90640d0, 95.94000d0, 98.90620d0, 101.0700d0,&
+     &102.9055d0, 106.4000d0, 107.8680d0, 112.4100d0, 114.8200d0,&
+     &118.6900d0, 121.7500d0, 127.6000d0, 126.9045d0, 131.3000d0,&
+     &132.9054d0, 137.3300d0,&
+     &138.91,140.12,140.91,144.24,147.00,150.36,151.97,157.25,&
+     &158.93,162.50,164.93,167.26,168.93,173.04,174.97,&
+     &178.4900d0, 180.9479d0,&
+     &183.8500d0, 186.2070d0, 190.2000d0, 192.2200d0, 195.0900d0,&
+     &196.9665d0, 200.5900d0, 204.3700d0, 207.2000d0, 208.9804d0,&
+     &209.,210.,222.,21*0.000d0/
```

Comment:
Any reason to duplicate the masses from `xtb_param_atomicmass`?

Reply:
now using `xtb_param_atomicmass`